test/testdata/fixtures/multi-dc-medusa/k8ssandra.yaml

```diff
@@ -33,11 +33,11 @@ spec:
     mgmtAPIHeap: 64Mi
   medusa:
     storageProperties:
-      storageProvider: local
+      storageProvider: s3_compatible
       bucketName: k8ssandra-medusa
       prefix: test
-      podStorage:
-        storageClassName: standard
-        accessModes:
-          - ReadWriteOnce
-        size: 100Mi
+      storageSecretRef:
+        name: medusa-bucket-key
+      host: test-hl.minio.svc.cluster.local
```

Comment:
Where does this service come from? The minio namespace already contains a minio-service which is created alongside the deployment, which might be better to use here to avoid confusion.